sample/src/main/java/com/microsoft/gctoolkit/sample/Main.java

```diff
@@ -4,18 +4,16 @@
 import com.microsoft.gctoolkit.io.GCLogFile;
 import com.microsoft.gctoolkit.io.SingleGCLogFile;
 import com.microsoft.gctoolkit.jvm.JavaVirtualMachine;
-import com.microsoft.gctoolkit.sample.aggregation.HeapOccupancyAfterCollectionAggregation;
 import com.microsoft.gctoolkit.sample.aggregation.HeapOccupancyAfterCollectionSummary;
 
-import java.nio.file.Files;
 import java.nio.file.Path;
 
 public class Main {
 
-    public static void main(String[] args) throws Exception {
+    public static void main(String[] args) {
```

Comment:
I think Exception was being thrown deliberately here.

Reply:
Since there aren't any checked exceptions thrown here, I think remove the throws clause is fine.